internal/nostr/nostr.go

```diff
@@ -804,7 +792,7 @@ func (svc *Service) publishRequestEvent(ctx context.Context, subscription *Subsc
 			"client_pubkey":    clientPubkey,
 		}).Error("Failed to publish to relay")
 		subscription.RequestEvent.State = REQUEST_EVENT_PUBLISH_FAILED
-		sub.Unsub()
+		relaySubscription.Unsub()
```

Comment:
should stopSubscription be called here? or not?

Reply:
Did that because it's directly available, but good to keep it consistent :+1: will change to stopSubscription